src/quera_ahs_utils/plotting.py

```diff
@@ -248,3 +244,182 @@ def plot_avg_density(densities, register, with_labels = True, custom_axes = None
     else:
         return None,ax
 
+
+
+#%%
+def visualize_UDG(positions,
+                  radii,
+                  ax=None,
+                  label_nodes=False,
+                  draw_disks=False,
+                  color_nodes=3,
+                  color_edges=0,
+                  scale=1,
+                  vertex_edge = False):
+    '''
+    Draws a unit disk graph.
```

Comment:
Can you update the doc-string to be consistent with the other functions? This will most likely break the documentation.

Reply:
Another option is to make this function private and not cover it in the documentation.